### PR TITLE
feat: register multiple users and orgs

### PR DESCRIPTION
**Pull request by rephus**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes:  #2094

### Description:
 register multiple users and orgs 

### Preview:

Peek 2022-05-13 10-31

Screenshot from 2022-05-13 10-31-28
Screenshot from 2022-05-13 10-31-40

I created a few other users in between...
Screenshot from 2022-05-13 11-17-37

### Scope of the changes (select all that apply):
- [x] Frontend  
- [x] Backend  
- [x] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

**Comment by rephus** [comment]
On render you can't test it because by default the flag is set to FALSE... you might want to test it locally 
image

File: docker/.env [hunk 1]
```diff
@@ -29,3 +29,5 @@ LIGHTDASH_CONFIG_FILE=/usr/app/lightdash.yml
 AUTH_GOOGLE_OAUTH2_CLIENT_ID=
 AUTH_GOOGLE_OAUTH2_CLIENT_SECRET=
 AUTH_DISABLE_PASSWORD_AUTHENTICATION=false
+
+ALLOW_MULTIPLE_ORGS=false
```
Comment (ZeRego) [inline docker/.env:33]: I think we should have this enabled on dev mode. Is going to be super useful :) 
And perhaps for PR envs as well so we can easily test it. I can't find reasons why not too. 🤔

**Review by ZeRego — APPROVED** [review]
lgtm 🎉 
Just think we could enable it on dev and PRs.

File: packages/frontend/src/pages/Login.tsx [hunk 2]
```diff
@@ -79,17 +79,6 @@ const Login: FC = () => {
         return <PageSpinner />;
     }
 
-    if (health.status === 'success' && health.data?.needsSetup) {
```
Comment (owlas) [inline packages/frontend/src/pages/Login.tsx:82]: @rephus @ZeRego isn't this still useful? It forced a redirect to the `register` screen if there is no existing user in the database. 

Still makes sense to me!

**Review by owlas — CHANGES_REQUESTED** [review]

File: packages/frontend/src/pages/Login.tsx (hunk 2 above)
Comment (rephus) [reply packages/frontend/src/pages/Login.tsx:82]: is it worth keeping the needsSetup flag just for that ?

Comment (rephus) [reply packages/frontend/src/pages/Login.tsx:82]: if we still want to keep it, I'd remate it to `hasUsers` or something more meaningful

Comment (ZeRego) [reply packages/frontend/src/pages/Login.tsx:82]: I don't think it makes sense the same way we show the register page even if they aren't allowed to register a new org. 🤔  Why/how would they end up in the login page before register an org ? 🤔

**Review by owlas — APPROVED** [review]

**Pull request merged**